Hyper/Meshes/Mesh.cs

```diff
@@ -20,18 +23,27 @@ internal class Mesh : IDisposable
 
     protected int NumberOfVertices;
 
+    private bool createdVertexArrayObject;
```

Comment:
_

Reply:
also maybe change int vaoId -> int? vaoId and just check if its null instead of this

Reply:
opengl does not like nullables